Gemfile

```diff
@@ -0,0 +1,11 @@
+source 'https://rubygems.org'
+
+gem 'github-pages', '>=207'
+
```

Comment:
```suggestion
gem 'webrick'
```

Not sure if it's a ruby version thing, but without this I can't run `bundle exec jekyll serve`

Reply:
added it back in now, thanks